e2e/create-pvc_test.go

```diff
@@ -173,6 +173,70 @@ var _ = Describe("CreatePVC", func() {
 		deletePod(tenantClient.CoreV1(), namespace, writerPod.Name)
 	})
 
+	It("should creates a RW-Many block pvc, with FS infra storage class, and attaches to pod", Label("pvcCreation", "RWX", "Block", "infra-FS"), func() {
+		pvcName := "test-pvc"
+		storageClassName := "infra-fs"
+		pvc := pvcSpec(pvcName, storageClassName, "10Mi")
+		volumeMode := k8sv1.PersistentVolumeBlock
+		pvc.Spec.VolumeMode = &volumeMode
```

Comment:
Nit: maybe use ptr.To here instead of a separate variable?

Reply:
Thanks @awels . good point - fixed